storage/integration_test.go

```diff
@@ -804,14 +812,22 @@ func TestIntegration_ObjectReadChunksGRPC(t *testing.T) {
 	ctx := context.Background()
 	client, err := newClientWithGRPC(ctx)
 	if err != nil {
-		t.Error(err)
+		t.Fatal(err)
 	}
 	defer client.Close()
 
-	content := []byte("Hello, world this is a grpc request")
+	content := bytes.Repeat([]byte("a"), 4<<10)
```

Comment:
Yep, this is what I had in mind as far as using bytes.Repeat-- maybe add a comment noting how large this file is and why it is this size.

Reply:
Done! I do think we need an even bigger blob though...this isn't big enough atm to trigger backend chunking on read (i.e. multiple stream Recv calls)

Reply:
Yep, let's do something like 5MB maybe? The test should still be pretty quick.

Reply:
Done and it works it seems! Test data is ~5MB and reading it with a ~4KB window.